src/components/Footer.test.js

```diff
@@ -0,0 +1,25 @@
+// Imports React into our test file.
+import React from 'react'
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import Footer from './Footer.js'
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter() })
+
+describe("When Footer renders", ()=>{
+    let footerRender
+    beforeEach(() => {
+        footerRender = shallow(<Footer />)
+    })
+    it("displays 'Footer', () => {
+        const footerRender = footerRender.find("h3") 
+        expect(footerRender.length).toEqual(3)
```

Comment:
Just one more thing. The expect is currently saying there should be 3 sets of `h3` tags. You currently only have one on your Footer.js. 
Please update the expect statement to reflect `1` on Footer.test.js
or
Update Footer.js to have `3` sets of h3 tags.